OpenRA.Mods.Common/Traits/Husk.cs

```diff
@@ -128,10 +132,14 @@ void INotifyRemovedFromWorld.RemovedFromWorld(Actor self)
 			self.World.RemoveFromMaps(self, this);
 		}
 
-		public void ModifyDeathActorInit(Actor self, TypeDictionary init)
+		void IDeathActorInitModifier.ModifyDeathActorInit(Actor self, TypeDictionary init)
 		{
 			init.Add(new FacingInit(Facing));
 		}
+
+		// We return self.Owner if there's no effective owner
+		bool IEffectiveOwner.Disguised { get { return true; } }
```

Comment:
I'm not sure I like the implications of "husks are always disguised".

Reply:
See https://github.com/OpenRA/OpenRA/pull/13276#discussion_r146107838 (and IRC).